assets/info-page/index.html

```diff
@@ -50,9 +50,9 @@
       function choosePrimaryPlatform() {
         const serverURL = "{{.ServerURL}}";
         const apiEndpoint = "{{.APIEndPoint}}";
-        const queryParam = "{{.QueryParamPrimaryPlatform}}";        
+        const queryParam = "{{.QueryParamPrimaryPlatform}}";
         fetch(`${serverURL}${apiEndpoint}?${queryParam}=${primaryPlatform}`)
-        .then(() => window.close());
+        .then(closeInfoPage());
```

Comment:
See above, but `closeInfoPage` also tries `window.close` (even though it likely won't work).